### PR TITLE
core/scripts/keystone improvements

### PR DESCRIPTION
**Pull request by archseer**
- Fixes a couple bugs (hostname not being parsed correctly, wrong plugin type)
- Adds commands for submitting and deleting workflows
- Allows specifying publickeys/nodelist/artifacts locations so we can use Aptos ones from a different directory

### CODE REVIEW

File: core/scripts/keystone/src/06_deploy_workflows_cmd.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		fs.Usage()
+		os.Exit(1)
+	}
+	fmt.Println("Deploying workflows")
+
+	// use a separate list
+	nodes := downloadNodeAPICredentials(".cache/NodeList.local.txt")
+
+	if _, err = os.Stat(*workflowFile); err != nil {
+		PanicErr(errors.New("toml file does not exist"))
```
Comment (HenryNguyen5) [inline core/scripts/keystone/src/06_deploy_workflows_cmd.go:43]: This should be `workflow file does not exist`, right?

File: core/scripts/keystone/src/06_deploy_workflows_cmd.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	workflowFile := fs.String("workflow", "workflow.yml", "path to workflow file")
+	err := fs.Parse(args)
+	if err != nil || workflowFile == nil || *workflowFile == "" {
+		fs.Usage()
+		os.Exit(1)
+	}
+	fmt.Println("Deploying workflows")
+
+	// use a separate list
+	nodes := downloadNodeAPICredentials(".cache/NodeList.local.txt")
```
Comment (HenryNguyen5) [inline core/scripts/keystone/src/06_deploy_workflows_cmd.go]: Why do we use a separate list here?

File: core/scripts/keystone/src/07_delete_workflows_cmd.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		fileFs := flag.NewFlagSet("test", flag.ExitOnError)
+		err = client.ListJobs(cli.NewContext(app, fileFs, nil))
+		helpers.PanicErr(err)
+
+		var parsed []JobSpec
+		err = json.Unmarshal(output.Bytes(), &parsed)
+		helpers.PanicErr(err)
+
+		for _, jobSpec := range parsed {
+			if jobSpec.WorkflowSpec.WorkflowID != "" {
```
Comment (HenryNguyen5) [inline core/scripts/keystone/src/07_delete_workflows_cmd.go:62]: When can this condition occur?

**Review by HenryNguyen5 — DISMISSED** [review]

File: core/scripts/keystone/src/07_delete_workflows_cmd.go (hunk 3 above)
Comment (archseer) [reply core/scripts/keystone/src/07_delete_workflows_cmd.go:62]: If you have a regular jobspec, we're trying to only delete workflows

**Review by DeividasK — APPROVED** [review]

**Review by krebernisak — APPROVED** [review]

**Pull request merged**